collectors/mimecast/collector.js

```diff
@@ -126,14 +127,26 @@ class MimecastCollector extends PawsCollector {
                 if (error.code) {
                     error.errorCode = error.code;
                 }
-                return callback(error);
+                if (error.statusCode && error.statusCode == 429) {
+                    error.errorCode = error.statusCode;
```

Comment:
not sure why we need this assignment because we don't pass this `error` object anywhere else.